doozer/doozerlib/cli/images_okd.py

```
@@ -669,100 +669,100 @@ def extract_stage_names(dfp: DockerfileParser) -> List[str]:
                     if tag not in payload_tag_names:
-                        detail = 'not in ART built OKD tags'
+                        detail = "not in ART built OKD tags"
 
-                    if os.path.exists(f'{temp_dir}/centos-release'):
+                    if os.path.exists(f"{temp_dir}/centos-release"):
                         scos_status = "Yes"
                     else:
                         if tag not in payload_tag_names:
                             scos_status = "N/A?"
 
                 except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)